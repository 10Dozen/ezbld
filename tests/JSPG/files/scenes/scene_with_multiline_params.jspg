# SceneWithMultilineParams

**pre_exec:
    MyGame.PreExec()
    if (MyGame.State() == States.OK) {
        MyGame.Activate()
    } else {
        MyGame.VerifyLoadedFile()
    }
**post_exec:
    MyGame.SetSceneResult(1, true)
    JSPG.GoTo('Scene3')
**

**goto:
    return MyGame.SelectScene()
*


Some description line.
