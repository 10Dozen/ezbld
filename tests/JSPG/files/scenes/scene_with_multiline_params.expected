[0]
cls = JSPGScene
name = SceneWithMultilineParams
pre_exec =
    \tMyGame.PreExec()
    \tif (MyGame.State() == States.OK) {
    \t    MyGame.Activate()
    \t} else {
    \t    MyGame.VerifyLoadedFile()
    \t}
post_exec =
    \tMyGame.SetSceneResult(1, true)
    \tJSPG.GoTo('Scene3')
goto =
    \treturn MyGame.SelectScene()

desc = Some description line.
